Library/Homebrew/test/cmd/mark_spec.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+require "cmd/mark"
+require "cmd/shared_examples/args_parse"
+require "tab"
+
+RSpec.describe Homebrew::Cmd::Mark do
+  it_behaves_like "parseable arguments"
+
+  describe "integration tests", :integration_test do
```

Comment:
Let's make this only one integration test, please, as they are super slow.

Ok to have more unit tests if desired.